egs/ami/s5b/local/chain/tuning/run_tdnn_lstm_1l.sh

```diff
@@ -5,11 +5,14 @@
 #System               tdnn_lstm1i_sp_bi_ld5 tdnn_lstm1l_sp_bi_ld5
 #WER on dev        20.6      19.8
 #WER on eval        20.1      19.2
-#Final train prob      -0.045 -0.067
```

Comment:
please give a bit more context in all of these scripts, at the top e.g. in this case where it says
` # same as 1i but with per-frame dropout on LSTM layer`
about what this is, e.g....
` # This (1l.sh) is thesame as 1i but with per-frame dropout on LSTM layer
  # It is a regular (not-fast) LSTM with per-frame dropout on [which gates?].`
And explain how it relates to the paper, e.g. is it "place4" in the paper?
You can send me the final pdf to put on my publications page if it's not already there,
and include a link from the script to the paper.